qemu/deps/playbook.yml

```diff
@@ -110,3 +117,9 @@
       shell:
         echo "Run test case failed"
       failed_when: run_case_result is failed
+
+    - name: Report test error
+      fail:
+        msg: "Some test cases are cancelled."
+      when:
+        cancelled_case > "0"
```

Comment:
type(cancelled_case) = int, so `"0"` should not be a string
another question, do we need to handle skipped cases?

Reply:
Fixed type issue. 

So far we don't have skipped cases, if need we could add handle skipped case.